src/pages/public_registration/PublicFilesContent.tsx

```diff
@@ -0,0 +1,146 @@
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import ButtonWithProgress from '../../components/ButtonWithProgress';
+import {
+  Button,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@material-ui/core';
+import { PublicRegistrationContentProps } from './PublicRegistrationContent';
+
+const StyledTableCell = styled(TableCell)`
+  min-width: 12rem;
+  font-size: 1rem;
+  word-wrap: break-word;
+`;
+
+const StyledNameTableCell = styled(StyledTableCell)`
+  font-weight: 700;
+  max-width: 35rem;
+`;
+
+const StyledLicenseImg = styled.img`
+  cursor: pointer;
+`;
+
+const StyledFilesContent = styled.div``;
+
+const PublicFilesContent = ({ registration }: PublicRegistrationContentProps) => {
+  const { t } = useTranslation('common');
+
+  const publiclyAvailableFiles = registration.fileSet.files.filter((file) => !file.administrativeAgreement);
+
+  return (
+    <StyledFilesContent>
+      <Typography variant="h4" component="h2" gutterBottom>
+        {t('registration:files_and_license.files')}
+      </Typography>
+
+      <TableContainer>
+        <Table size="small">
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>{t('registration:files_and_license.title')}</StyledTableCell>
+              <StyledTableCell>{t('size')}</StyledTableCell>
+              <StyledTableCell>{t('version')}</StyledTableCell>
+              <StyledTableCell>{t('registration:files_and_license.license')}</StyledTableCell>
+              <StyledTableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {publiclyAvailableFiles.map((file) => (
+              <FileRow file={file} />
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </StyledFilesContent>
+  );
+};
+
+const FileRow = ({ file }: { file: File }) => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation('common');
+  const { identifier } = useParams<{ identifier: string }>();
+  const [isLoadingFile, setIsLoadingFile] = useState(false);
+  const [currentFileUrl, setCurrentFileUrl] = useState('');
+
+  const handleDownload = async (fileId: string) => {
+    setIsLoadingFile(true);
+    const file = await downloadFile(identifier, fileId);
+    if (!file || file?.error) {
+      dispatch(setNotification(file.error, NotificationVariant.Error));
+    } else {
+      setCurrentFileUrl(file);
+    }
+    setIsLoadingFile(false);
+  };
+
+  const licenseData = licenses.find((license) => license.identifier === file.license?.identifier);
+  const fileEmbargoDate = file.embargoDate ? new Date(file.embargoDate) : null;
+
+  return (
+    <TableRow key={file.identifier} hover>
+      <StyledNameTableCell>{file.name}</StyledNameTableCell>
+      <StyledTableCell>{Math.round(file.size / 1000)} kB</StyledTableCell>
```

Comment:
Syns det er et lite nederlag hver gang vi tar inn en ny pakke, men lurer på om vi bare skal ta inn denne for å vise Bytes på en god måte: https://www.npmjs.com/package/pretty-bytes
Kan brukes både her og ved filopplasting

Reply:
Ganske liten størrelse på pakken og 0 dependencies, så det går nok fint ja 👍🏼